internal/perms/acl.go

```diff
@@ -209,6 +222,64 @@ func (a ACL) Allowed(r Resource, aType action.Type, userId string, opt ...Option
 	return
 }
 
+// ListPermissions builds a set of Permissions based on the grants in the ACL for the List action, for the requested scopes and resource type.
+func (a ACL) ListPermissions(requestedScopes map[string]*scopes.ScopeInfo, requestedType resource.Type, requestedActions action.ActionSet) []Permission {
+	perms := make([]Permission, 0, len(requestedScopes))
+	for scopeId := range requestedScopes {
+		p := Permission{ScopeId: scopeId}
```

Comment:
```suggestion
		p := Permission{
			ScopeId: scopeId,
			Resource: requestedType,
			Action: action.List,
		}
```

Reply:
Ha, good catch, thanks! Addressed in [e854b6b](https://github.com/hashicorp/boundary/pull/2335/commits/e854b6beaff280ea7e1db104a0f6da8a01aceb89)